include/slam_toolbox/merge_maps_kinematic.hpp

```diff
@@ -48,37 +48,47 @@
 #include <unistd.h>
 #include <fstream>
 
-#include <boost/archive/text_oarchive.hpp>
-#include <boost/archive/text_iarchive.hpp>
-#include <boost/serialization/vector.hpp>
-#include <boost/serialization/list.hpp>
-#include <boost/serialization/string.hpp>
-
 #define MAP_IDX(sx, i, j) ((sx) * (j) + (i))
 
-class MergeMapTool
+class MergeMapsKinematic
 {
 
 public:
-  MergeMapTool();
-  ~MergeMapTool();
+  MergeMapsKinematic();
+  ~MergeMapsKinematic();
 
 private:
+
   // setup
   void SetConfigs();
 
-  // callbacks
+  // callback
   bool MergeMapCallback(slam_toolbox::MergeMaps::Request  &req, slam_toolbox::MergeMaps::Response &resp);
   bool AddSubmapCallback(slam_toolbox::AddSubmap::Request  &req, slam_toolbox::AddSubmap::Response &resp);
   void ProcessInteractiveFeedback(const visualization_msgs::InteractiveMarkerFeedbackConstPtr& feedback);
-  void KartoToROSOccupancyGrid(const karto::LocalizedRangeScanVector& scans);
+  void KartoToROSOccupancyGrid(const karto::LocalizedRangeScanVector& scans, nav_msgs::GetMap::Response& map);
 
-  inline bool FileExists(const std::string& name)
+  karto::Pose2 ApplyCorrection(const karto::Pose2& pose, const tf::Transform& submap_correction)
```

Comment:
Why are these defined in the header?

Reply:
because there are 2 for different data types so i thought it would be nicer to have them in the header, I can return them to the source file

Reply:
please

Reply:
^ plz move <3